src/app/child-dev-project/notes/model/note.ts

```diff
@@ -149,6 +149,9 @@ export class Note extends Entity {
       attendance = new EventAttendance();
       this.childrenAttendance.set(childId, attendance);
     }
+    if (!(attendance instanceof EventAttendance)) {
+      attendance = Object.assign(new EventAttendance(), attendance);
```

Comment:
This creates a bug. After recording the attendance once, it is not possible to change the attendance in the `NoteDetailsComponent` or the `RollCallComoponent`. I think because of the `Object.assign` call, the attendance object is a different one than in the `childrenAttendance` map and therefore editing this new object is not persisted in the entity.